source/_integrations/husqvarna_automower.markdown

```diff
@@ -80,6 +82,17 @@ The My Home Assistant redirect feature needs to be setup to redirect to your Hom
 
 Once you have enabled the Husqvarna Automower integration, you should see the following entities:
 
+### Binary Sensor
+
+The integration will create the following binary sensors:
+
+- Battery charging  
+  *The mower is currently charging. Note that the mower will only report the activity of charging if it was its own decision to go home and charge. For example, if it wants to mow but is charging because the battery is low. If the mower is restricted (for example, due to the weekly schedule), it may charge when in the charging station, but the binary sensor will not show.*
```

Comment:
:smile: this sounds a bit like a mower with consciousness, with its own will, wants and needs. 
What does it mean:
- *its own decision*
- *it wants to mow*

Reply:
Lol. This is from the official API documentation.
It just means, that it's only reported, when the mower is in a regular operation cycle (mowing/charging/mowing/...) and is not forced to stay in the dock, by the user.
But I can improve the deception.

Reply:
yes, I think a little rephrase would help.